app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -14,7 +14,7 @@ export default class TidyTree {
         addOverlays: true,
         attribute: "aggregatescore",
         leafNodeHeight: 35,
-        minWidth: 1000,
+        minWidth: 960,
```

Comment:
This was necessary to prevent default scrolling on first load because the container maxes out at 1200px